server/fleet/software.go

```diff
@@ -21,6 +21,9 @@ type Software struct {
 	GenerateCPE string `json:"generated_cpe" db:"generated_cpe"`
 	// Vulnerabilities lists all the found CVEs for the CPE
 	Vulnerabilities VulnerabilitiesSlice `json:"vulnerabilities"`
+
+	// HostCount is the amount of hosts that currently have this software installed
+	HostCount int `json:"host_count" db:"host_count"`
```

Comment:
https://github.com/fleetdm/fleet/issues/2049#issuecomment-951173380 mentions `hosts_count` (just FYI).

Reply:
Ah, that's right. Updating!

Reply:
We do use the `host_count` to define the amount of hosts for something (eg: labels), so this is more standard. What do you think?